api/python/tests/test_api.py

```diff
@@ -87,3 +88,42 @@ def test_put_copy_get(self):
 
         assert np.array_equal(data, data2)
         assert meta == meta2
+
+    @patch('t4.session.get_session')
+    @responses.activate
+    def test_login(self, get_session):
+        mock_session = Mock()
+        get_session.return_value = mock_session
+
+        mock_response = Mock()
+        mock_response.ok = True
+        mock_session.post.return_value = mock_response
+
+        mock_creds_response = Mock()
+        creds_data = {
+            'AccessKeyId': 'asdf',
+            'SecretAccessKey': 'asdf',
+            'SessionToken': 'asdf',
+            'Expiration': datetime.utcnow().isoformat() + '-02:00' # so it's not expired yet
```

Comment:
I think this is futzing the timezone, it's probably clearer to do something like
```
+ timedelta(hours=2)
```
then format it to an iso string